src/main/java/teammates/ui/controller/Action.java

```diff
@@ -367,21 +367,19 @@ public ActionResult executeAndPostProcess() throws EntityDoesNotExistException {
         return response;
     }
 
+    /**
+     * Adds the list of status messages generated after performing an action to session.
```

Comment:
Rephrase this? Adds the list of status messages from ActionResult intto session variables.